dpgen/auto_test/EOS.py

```diff
@@ -117,6 +119,7 @@ def make_confs(self,
 
             else:
                 print('gen eos from ' + str(self.vol_start) + ' to ' + str(self.vol_end) + ' by every ' + str(self.vol_step))
+                if self.vol_abs : print('(absolute volume)')
```

Comment:
For suggestion 1, since the user has already push the "vol_abs" button in property.json, maybe it's ok to be in "Description" part.
Suggestion 2 is implemented.